djsf.json

```diff
@@ -37,6 +37,11 @@
 					"description": "The type of the generated schedule (\"period\" or \"day\").",
 					"id": "#scheduletype",
 					"required":true
+				},
+				"timestamp": {
+					"type":"string",
+					"required":true,
+					"description": "The timestamp when the period schedule is generated."

```

Comment:
Please also document that this timestamp is in ISO 8601 format and add an example.